pkgs/net-doc/net/scribblings/http-client.scrbl

```diff
@@ -204,13 +204,18 @@ response, which is why there is no @racket[#:closed?] argument like
  The function returns four values:
 
  @itemize[
-   @item{If @racket[ssl?] was @racket[#f] then @racket[#f]. Otherwise
-     an @racket[ssl-client-context?] that has been negotiated with the
-     target.
-
-     If @racket[ssl?] was a protocol symbol, then a new
-     @racket[ssl-client-context?] is created, otherwise the current
-     value of @racket[ssl?] is used.}
+   @item{
+     If @racket[ssl?] is @racket[#f], returns @racket[#f], otherwise
```

Comment:
It's kinda weird to say "returns" here. I like the old text better. The first bullet point is about the first value of the four values. There's no function that returns this single value specifically.

Reply:
The original phrasing read like an `if` expression, not an *English* sentence.  It took me some puzzling to understand what it was talking about.  A better phrasing is welcome, but it should read like an Enflish sentence.

Reply:
Perhaps replace "returns ..." with "the (first) value is ..."

Reply:
Rephrased this part (actually, the whole list).